js/components/navigationBar.js

```diff
@@ -91,11 +91,7 @@ class NavigationBar extends ImmutableComponent {
 
   get bookmarked () {
     return this.props.activeFrameKey !== undefined &&
-      siteUtil.isSiteBookmarked(this.props.sites, Immutable.fromJS({
-        location: this.props.location,
-        partitionNumber: this.props.partitionNumber,
-        title: this.props.title
-      }))
+      siteUtil.isSiteBookmarked(this.props.sites, Immutable.fromJS({location: this.props.location}))
```

Comment:
we differentiate on bookmarks with different partitions, at least until today with previous releases.  For example a user might want to bookmark twitter.com when logged in as user A and twitter.com when logged in as user B.